stock_brand/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+#  Copyright (C) 2019 Sunflower IT <sunflowerweb.nl>
+#  License GNU Affero General Public License <http://www.gnu.org/licenses/>.
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.multi
+    def action_confirm(self):
+        ret = super(SaleOrder, self).action_confirm()
+        for this in self:
+            for picking in this.picking_ids:
```

Comment:
no need for loop here.
```suggestion
            order.picking_ids.write({'brand_id': order.brand_id})
```